src/pages/AnimeKO/style.less

```diff
@@ -0,0 +1,25 @@
+@import './../pages';
+
```

Comment:
The episode highlighting is quite ugly

![image](https://github.com/MALSync/MALSync/assets/12599334/a9104b72-9fe9-4290-947b-1c102609d724)